mscore/plugin/api/elements.h

```diff
@@ -22,6 +22,9 @@
 #include "libmscore/notedot.h"
 #include "libmscore/segment.h"
 #include "libmscore/accidental.h"
+#include "libmscore/museScoreCore.h"
```

Comment:
Doesn't exist

Reply:
Dang!

I got burned by the 'ole Windows case insensitivity "feature"!

-Dale

Reply:
;-)